roles/test-provision-apb/tasks/main.yml

```diff
@@ -21,7 +18,4 @@
   register: webpage
   retries: 10
   delay: 20
-  failed_when:
-    - webpage.status == 403
-    - '"kc_logged_in_users" not in webpage.content'
```

Comment:
@josemigallas renamed those to `keycloak_`. We need to either standatize the metrics SPI soon or work to get metrics into upstream.